### PR TITLE
Formatting all sourcefiles

### PR DESCRIPTION
**Pull request by sveisvei**
* script `scripts/format-stylesheet.js` output

### CODE REVIEW

File: src/styles/core/form/form_elements.css [hunk 1]
```diff
@@ -486,74 +433,61 @@ select.blank{
     text-shadow: none;
     box-shadow: none;
     margin: 0;
-    -webkit-appearance: none;
```
Comment (antidecaf) [inline src/styles/core/form/form_elements.css]: appearance still needs the vendor prefix.

File: src/styles/core/form/form_elements.css [hunk 2]
```diff
@@ -486,74 +433,61 @@ select.blank{
...
+    width: 20px;
+}
+.customcbx input[type='checkbox'] + p.decoration { background-position: -25px -262px; }
+.customcbx input[type='checkbox']:checked + p.decoration { background-position: -25px -287px; }
+.customcbx input[type='radio'] + p.decoration { background-position: -52px -262px; }
+.customcbx input[type='radio']:checked + p.decoration { background-position: -52px -287px; }
+/* ----- Custom dropdown ------------------------------------ */
+.custom-dropdown select {
+    /* tar bort default appearance */
     appearance: none;
```
Comment (antidecaf) [inline src/styles/core/form/form_elements.css]: appearance still needs the vendor prefix

File: src/styles/core/form/form_elements.css (hunk 1 above)
Comment (sveisvei) [inline src/styles/core/form/form_elements.css]: This is intended, autoprefixer will add prefixes based on browser targets.

File: src/styles/core/form/form_elements.css (hunk 2 above)
Comment (sveisvei) [inline src/styles/core/form/form_elements.css]: autoprefixer fixes in build. https://github.com/finn-no/spaden/pull/32#discussion_r31520125

**Pull request closed without merging**